lerobot/common/datasets/push_dataset_to_hub/codebase_version.md

```diff
@@ -0,0 +1,58 @@
+## Using / Updating `CODEBASE_VERSION` (for maintainers)
```

Comment:
Could we rename the file in ALL CAPS by convention?

Reply:
Fixed https://github.com/huggingface/lerobot/pull/318/commits/71a9ec33ca05a6fd0833d4221913463ca46cbb07
Btw, did you know [this](https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git)? ^^